### DISCUSSION TITLE
Caliptra LMS mailbox command

### DISCUSSION THREAD

**Discussion by ChiaweiW** [chipsalliance/caliptra-sw#1374] [General]
Hi Team,

Caliptra 1.0 supports the LMS algorithm for post-quantum cryptography needs.
However, currently the LMS verification is only applied on Caliptra FW images.

Is it possible to export the LMS feature as a mailbox command service like ECDSA384_SIGNATURE_VERIFY do?
Thus SoC with Caliptra integrated can also leverage this PQC benefit.

**Comment by ericeilertson** [comment]
We will add this capability to the Caliptra 1.1 firmware.  The parameters supported will be sha256/192, w=4, and h=15.

**Reply by ChiaweiW** [reply]
Thanks for the feedback!

In addition, we are on the way to finalize our HW design with Caliptra integrated.
Will 1.1 RTL be required to execute the 1.1 FW?

**Reply by ericeilertson** [reply]
To use the 1.1 firmware you do not need to use 1.1 RTL.

**Reply by ChiaweiW** [reply]
Thanks for the confirmation!
